modules/imgcodecs/src/loadsave.cpp

```diff
@@ -234,6 +234,7 @@ static ImageDecoder findDecoder( const String& filename ) {
     /// Open the file
     FILE* f= fopen( filename.c_str(), "rb" );
 
+    CV_LOG_IF_WARNING(NULL, !f, "image file could not be opened or path could not be found");
     /// in the event of a failure, return an empty image decoder
     if( !f )
```

Comment:
1.

> `LOG_IF`

No need to use condition here, if two lines below we have the same condition.
So it makes sense to move message there without condition.

---

2.
Check [this line](https://github.com/opencv/opencv/blob/4.5.4/modules/imgcodecs/src/loadsave.cpp#L771) and dump "filename" value too.

---
3.

Original problem of mentioned issue is missing validation for input parameters in `seamlessClone()` implementation.
Check previous discussions #20617 and add `CV_Assert(!_src.empty());` to resolve the mentioned issue.

Reply:
First two suggestions, has been applied to first commit and third suggestion has been added as a separate commit.